src/languages/java.md

```diff
@@ -1,3 +1,101 @@
+
+## Support build automation
+
+Platform.sh has support for software project management and comprehension tool in the Java World: 
+
+* [Gradle](https://gradle.org/)
+* [Maven](https://maven.apache.org/)
+* [Ant](https://ant.apache.org/)
+
+
+
+## Accessing services
+
+To access various [services](/configuration/services.md) with Java, see the following examples.  The individual service pages have more information on configuring each service.
+
+
+
+{% language name="MongoDB", type="php", url="https://examples.docs.platform.sh/php/mongodb" -%}
+
+{% language name="MySQL", type="php", url="https://examples.docs.platform.sh/php/mysql" -%}
+
+{% language name="PostgreSQL", type="php", url="https://examples.docs.platform.sh/php/postgresql" -%}
+
+{% language name="Redis", type="php", url="https://examples.docs.platform.sh/php/redis" -%}
+
+<!--
+
+{% language name="RabbitMQ", type="php", url="https://examples.docs.platform.sh/php/rabbitmq" -%}
+
+{% language name="Solr", type="php", url="https://examples.docs.platform.sh/php/solr" -%}
+
+{% codetabs name="Elasticsearch", type="php", url="https://examples.docs.platform.sh/php/elasticsearch" -%}
+
+{% language name="Memcached", type="php", url="https://examples.docs.platform.sh/php/memcached" -%}
+-->
+
+{%- endcodetabs %}
+
+
+
+## Project templates
+
+A number of project templates for major Java applications are available on GitHub. Not all of them are proactively maintained but all can be used as a starting point or reference for building your own website or web application.
+
+### Applications
+
+
+#### Maven Spring
+* [Spring Boot MySQL](https://github.com/platformsh/template-spring-boot-maven-mysql)
+* [Spring Boot MongoDB](https://github.com/platformsh/template-spring-mvc-maven-mongodb)
+<!--
+* [Spring Boot PostgreSQL](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Memcached](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Redis](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Elasticsearch](https://github.com/platformsh/platformsh-example-ezplatform)
+
+#### Gradle Spring
+
+* [Spring Boot MySQL](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot PostgreSQL](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot MongoDB](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Memcached](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Redis](https://github.com/platformsh/platformsh-example-ezplatform)
+* [Spring Boot Elasticsearch](https://github.com/platformsh/platformsh-example-ezplatform)
+
+
+#### Maven Jakarta
+* The same sample above (generic)
+* Apache TomEE
+* Payara
+* Thorntail
+
+#### Visual sample
+Spring MVC
+Jakarta EE and MVC SPEC
+
+
+-->
```

Comment:
Why the big commented out block?  If those aren't ready just don't include them at all yet.

Reply:
Done